utils/dbz_event_to_fhir_config.json

```diff
@@ -17,6 +17,74 @@
         "rest": "/ws/rest/v1/concept/{uuid}?v=full"
       }
     },
+
+    "concept_complex":{
```

Comment:
Why are these records added? I mean they don't seem to be relevant to FHIR resources (since they have no `fhir` field), are they?

Originally we were trying to reused the Atom Feed module config but now that we are diverging more and more, we should remove all non-FHIR records from this table (and don't add new ones).